web/javascript/common.js

```diff
@@ -61,7 +61,7 @@ $(document).ready(function() {
 			// Fix min height for isMobileDevice when run in full screen mode from home screen
 			// so the footer appears in the right place
 			$('body div#torrent_container').css('min-height', '338px');
-		$("label[for=torrent_upload_url]").text("URL: ");
+		$("label[for=torrent_upload_url]").text = "URL: ";
```

Comment:
This should be reverted, the previous version was correct (`.text()` is a jQuery method)

Reply:
Users have reported Safari being fixed by this change.

Reply:
This is quite obviously wrong.

Perhaps you're confused by a similar change on master? That one doesn't use jQuery so it doesn't apply here.

Reply:
I'm OK with all the other backport changes, but this doesn't actually seem to be a backport and I agree with @glebm that this change seems to be in error. [text(text)](http://api.jquery.com/text/#text2) is indeed a jQuery method that's used in [other parts of the 2.9x web client](https://github.com/transmission/transmission/blob/2.9x/web/javascript/transmission.js#L1266).

I'm :+1: on this PR if this one-liner is removed.

Reply:
Removed. I think whatever bug this fixes has long since been fixed on Apple's side.